source/libraries/ContractDeployer.ts

```diff
@@ -193,8 +196,9 @@ export class ContractDeployer {
     private async createGenesisUniverse(): Promise<ContractBlockchainData> {
         const delegatorBuilder = this.ethjsContract(this.signatures["Delegator"], this.bytecodes["Delegator"], { from: this.testAccounts[0], gas: this.gasAmount });
         const universeBuilder = this.ethjsContract(this.signatures["Universe"], this.bytecodes["Universe"], { from: this.testAccounts[0], gas: this.gasAmount });
-        const receiptAddress = await delegatorBuilder.new(this.controller.address, `0x${binascii.hexlify("Universe")}`);
-        const receipt = await this.ethjsQuery.getTransactionReceipt(receiptAddress);
+        const transactionHash = await delegatorBuilder.new(this.controller.address, `0x${binascii.hexlify("Universe")}`);
+        await waitForTransactionToBeSealed(this.ethjsQuery, transactionHash);
+        const receipt: ContractReceipt = await this.ethjsQuery.getTransactionReceipt(transactionHash);
```

Comment:
Style: Remove the type definition on this variable, it should be inferred from `getTransactionReceipt`. If it isn't, we should update the `.d.ts` file.